test/integration/features/cmd-docker-env.feature

```diff
@@ -0,0 +1,41 @@
+@cmd-docker-env @command
+Feature: Command docker-env
+Command docker-env helps the user to set environmental variables which are needed force
+the usage of Minishift with docker command on VM host.
+INFO: This feature runs against persistent shell instance. To use non-default shell, please
+select one from: bash, cmd, powershell, tcsh, zsh with TEST_WITH_BASH parameter of make integration.
+
+  Scenario: Docker client is not set up
+  INFO: This scenario starts interactive shell instance, which will be closed in the end of this feature.
+    Given user starts shell instance
+     When executing in shell command "docker info"
+     Then stdout of shell should not contain "Operating System: Minishift Boot2Docker ISO Version"
```

Comment:
so what should it contain then? Seems to be odd to test what should _not_ be there. Why do you expect this to be there?

Reply:
@hferentschik this is here to prevent potential situation when the one who runs it has the separate Minishift instance running and has Docker preconfigured via .bashrc, might then the docker info pass? Tried to be defensive here.